AnkiDroid/src/main/java/com/ichi2/anki/Preferences.java

```diff
@@ -185,13 +190,20 @@ public static Intent getPreferenceSubscreenIntent(Context context, String subscr
         i.putExtra(PreferenceActivity.EXTRA_NO_HEADERS, true);
         return i;
     }
-
     private void initSubscreen(String action, PreferenceContext listener) {
         PreferenceScreen screen;
         switch (action) {
             case "com.ichi2.anki.prefs.general":
                 listener.addPreferencesFromResource(R.xml.preferences_general);
                 screen = listener.getPreferenceScreen();
+                if(AdaptionUtil.hasReducedPreferences()) {
```

Comment:
```suggestion
                if (AdaptionUtil.hasReducedPreferences()) {
```
And this is the example I mention above of programmatically knocking out the preferences you don't want